user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -374,8 +374,7 @@ def infer_app_meta(eventlogs: List[str]) -> Mapping[str, Mapping]:
     # run any plugin hooks on profile_df
     for p in plugins:
         plugin = load_plugin(p)
-        if plugin:
-            profile_df = plugin.load_profiles_hook(profile_df)
+        profile_df = plugin.load_profiles_hook(profile_df)
```

Comment:
Why did you get rid of the plugin check?  That might be good to keep in case there is an issue with load_plugin().  Or add in a try block.